packages/zoltan2/src/input/Zoltan2_MeshAdapter.hpp

```diff
@@ -127,6 +127,7 @@ class MeshAdapter : public BaseAdapter<User> {
 
 #ifndef DOXYGEN_SHOULD_SKIP_THIS
   typedef typename InputTraits<User>::scalar_t scalar_t;
+  typedef typename InputTraits<User>::lno_t offset_t; // default to lno_t
```

Comment:
I think originally when I tried this as offset it was fine (compiled and tests ok). I will switch it over.